FSI/DisplacementDelta.C

```diff
@@ -37,10 +37,10 @@ void preciceAdapter::FSI::DisplacementDelta::initialize()
 void preciceAdapter::FSI::DisplacementDelta::write(double* buffer, bool meshConnectivity, const unsigned int dim)
```

Comment:
By the way, since this file is touched anyway: does solids4foam also support relative displacements (i.e., not to an absolute zero, but relatively to the previous state)?

Reply:
Documented for future work: https://github.com/precice/openfoam-adapter/issues/239